db/migrations/versions/~2024_04_22_1728-8e4d1e36ebbf_.py

```diff
@@ -0,0 +1,44 @@
+"""empty message
```

Comment:
Can we add a message as it shows up when you run the migrations pls?

Reply:
```suggestion
"""FS-4071: Add Boolean to determine whether we show email
or a link to the contact us page, differs per fund. 
```